Simplenote/SPNavigationController.swift

```diff
@@ -24,21 +26,23 @@ class SPNavigationController: NSViewController {
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-
+    
     override func loadView() {
         guard let initialViewController = topViewController else {
             fatalError()
         }
 
         view = NSView()
+        heightConstraint = view.heightAnchor.constraint(equalToConstant: .zero)
+        heightConstraint?.isActive = true
```

Comment:
Since we're now doing this, the `minimumHeightConstraint` workaround isn't required anymore. We can:

- Maybe convert this constraint into an IUO? or not?
- Move the `.activate` to L57 instead (where `miminumHeightConstraint` lives now)

**Also** I'm ... slightly worried about setting it to `.zero` by default. It is possible, at some point in time, the Window will actually be `.zero` height.